src/objects/core/zcl_abapgit_filename_logic.clas.testclasses.abap

```diff
@@ -505,7 +505,7 @@ CLASS ltcl_run_checks IMPLEMENTATION.
       iv_ext  = 'properties' ).
 
     cl_abap_unit_assert=>assert_equals(
-      exp = 'zif_abap.intf.i18n.en-GB.properties'
+      exp = 'zif_abap.intf.texts.en-GB.properties'
```

Comment:
I think interfaces don't even have text elements according to https://github.com/SAP/abap-file-formats/tree/main/file-formats/intf. So I would have expected that `zif_abap.intf.i18n.en-GB.properties` is correct?!